sound/soc/intel/boards/sof_rt5682.c

```diff
@@ -60,6 +63,16 @@ static int sof_rt5682_quirk_cb(const struct dmi_system_id *id)
 }
 
 static const struct dmi_system_id sof_rt5682_quirk_table[] = {
+	{
+		.callback = sof_rt5682_quirk_cb,
+		.matches = {
+			DMI_MATCH(DMI_SYS_VENDOR, "ADI"),
+			DMI_MATCH(DMI_PRODUCT_NAME, "Minnowboard Turbot"),
+		},
+		.driver_data = (void *)(SOF_RT5682_MCLK_EN |
+					SOF_RT5682_MCLK_BYT_EN |
+					SOF_RT5682_SSP_CODEC(2)),
+	},
```

Comment:
I would do this differently.
The MCLK should be on for BYT/CHT, except for the MinnowBoard B0 (see bytcr_rt5651) and Up board (not supported on the connector).